### ISSUE TITLE
bug array $config['skin_logo'] does not work (fix included)

### ISSUE THREAD

**Issue by mrAceT** [roundcube/roundcubemail#6330]
I discovered that $this->template_name in the 'logo routine' always contains 'mail' but the config value 'skin' does contain the right value..

file: program\include\rcmail_output_html.php (version 1.3.6)
line 1160 add: (after "if (is_array($logo)) {")
`$template_name = $this->config->get('skin', 'classic');`

alter this line (just below previous mentioned line 1160)
`if ($template_logo = $logo[$this->template_name]) {`
alter it to:
`if ($template_logo = $logo[$template_name]) {`

now one can configure a logo for each skin an use the intended '*' as a fall back:

```
$config['skin_logo'] = array('*'       => './logo.png'
			,'larry'   => './logo.png'
			,'classic' => './logo-2.png'
			);
```

**Comment by johndoh** [comment]
It does not always contain `mail`. `mail` is the name of one of the templates. In 1.3.6 and earlier the `skin_logo` config option can be used to set different logos for different templates, not skins. For example a different logo for the `messageprint` template.

Changes have just been merged into git-master (4ea790b) which extend this config option to allow different logos per skin as well as per template. Hopefully this will be included in 1.4

**Comment by mrAceT** [comment]
Ah the difference between "skin" an "template". I understand. So I not so much fixed a bug, as I added a feature ;)
